tests/pipelines/kit/etcd-list-job-pipeline.yaml

```diff
@@ -0,0 +1,138 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Pipeline
+metadata:
+  name: etcd-list-job-pipeline
+  namespace: tekton-pipelines
+spec:
+  params:
+  - name: name
+    description: The name of the test cluster.
+    default: "guest"
+  - name: scheduler-replicas
+    default: "1"
+    description: Number of scheduler replicas
+  - name: controller-manager-replicas
+    default: "1"
+    description: Number of controller manager replicas
+  - name: apiserver-replicas
+    default: "1"
+    description: Number of APIserver replicas
+  - name: apiserver-image
+    default: "public.ecr.aws/eks-distro/kubernetes/kube-apiserver:v1.19.13-eks-1-19-9"
+    description: Image of apiserver
+  - name: apiserver-parameters
+    default: "[]"
+    description: Parameters of the apiserver container
+  - name: apiserver-instance-type
+    default: "m5.16xlarge"
+    description: Instance type for the apiserver
+  - name: etcd-replicas
+    default: "3"
+    description: Number of ETCD replicas
+  - name: etcd-image
+    default: "public.ecr.aws/eks-distro/etcd-io/etcd:v3.4.16-eks-1-21-4"
+    description: Image of ETCD
+  - name: etcd-parameters
+    default: "[]"
+    description: Parameters of the ETCD container
+  - name: etcd-instance-type
+    default: "m5.16xlarge"
+    description: Instance type for the ETCD
+  - name: kubernetes-version
+    default: "1.19"
+    description: Kubernetes version for the guest cluster
+  - name: substrate-name
+    description: Name of the substrate cluster
+  - name: node-count
+    default: "5"
+    description: Number of worker nodes
+  - name: number-of-pods
+    default: "50"
+    description: Number of dummy pods
+  - name: pod-size
+    default: "5000"
+    description: Size of the pod object
+  - name: parallelism
+    default: "11"
+    description: K8S job parallelism
+  - name: etcd-list-job-image
+    default: "285627164574.dkr.ecr.us-west-2.amazonaws.com/etcd-load-tests:latest"
```

Comment:
I don't think we should add this particular test here as this is a very specific test cases needed by etcd team and here we want more generic tests

Reply:
agree. that's a mistake. I have removed it.